Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -237,6 +307,101 @@ def fetch_incidents(
     return next_run, incidents
 
 
+def set_alert_severity(orca_client: OrcaClient, args: Dict[str, Any]) -> CommandResults:
+    alert_id = args.get("alert_id")
+    score = args.get("score")
+
+    assert alert_id and score
+    demisto.debug(f"Set alert severity {alert_id=} {score=}")
+
+    response = orca_client.set_alert_score(alert_id=alert_id, orca_score=score)
+
+    context = {}
+    if "alert_id" in response:
+        context = {
+            "id": response.get("alert_id"),
+            "details": response.get("details", {}).get("description"),
+            "severity": response.get("details", {}).get("severity")
+        }
+    if "error" in response:
```

Comment:
We handle error here because api will return error if the alert have the same severity. 
Not sure that we should raise exception in this case.

Reply:
what is the message returned in the error from the response? i see why it's not considered as a failure, but we we're expecting to see?

Reply:
In case if the alert already have requested severity - response will have status code 400 and `error` will contains `requested to set same configuration`
By my opinion it is not error from the user perspective.

Reply:
Anyway i changed custom handler. Instead i just raise an DemistoException

Reply:
no, i agree that in this case it should not be considered as an error from the user perspective, so please add an if statement to check if this error is returned (the same state) then it should not throw an error just return a statement that the "the stats is ...."
otherwise, which's a real error, it should throw an exception.